src/Http/Controllers/SessionController.php

```diff
@@ -80,4 +118,14 @@ protected function generateAuthToken($user) {
 
         return $jwt->encode();
     }
+
+    protected function generateTokenAndSendResponse($user) {
+        if ($user) {
+            $token = $this->generateAuthToken($user);
+            Log::info("token: " . $token);
```

Comment:
Does not seem to be a good idea to me to log this token.

Reply:
yep, mistake